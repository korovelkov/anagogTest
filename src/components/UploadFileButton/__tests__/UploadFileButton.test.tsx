import React from 'react';
import Button from '../UploadFileButton';
import renderer from 'react-test-renderer';

it('render button', () => {
  const tree = renderer.create(<Button>test</Button>).toJSON();

  expect(tree).toMatchSnapshot();
});
