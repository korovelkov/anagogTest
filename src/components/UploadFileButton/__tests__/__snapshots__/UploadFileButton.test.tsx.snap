// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button 1`] = `
<button
  className="ant-btn UploadFileButton"
  onClick={[Function]}
  type="button"
>
  <span>
    test
  </span>
  <label>
    <input
      type="file"
    />
  </label>
</button>
`;
